src/controller/case.controller.js

```diff
@@ -43,11 +45,20 @@ export const caseStageController = async (request, h) => {
 
   await caseService.updateCaseStage(caseId, nextStage, request.db);
 
-  await publish(config.get("aws.caseStageUpdatedTopicArn"), {
-    caseRef: caseRecord.caseRef,
-    previousStage,
-    currentStage: nextStage
-  });
+  const event = {
+    id: randomUUID(),
+    source: config.get("serviceName"),
+    specVersion: "1.0",
+    type: `cloud.defra.${config.get("cdpEnvironment")}.${config.get("serviceName")}.case.stage.updated`,
+    data: {
+      caseRef: caseRecord.caseRef,
+      previousStage,
+      currentStage: nextStage
+    },
+    traceparent: getTraceId()
```

Comment:
All events should include the `traceparent`. Can we add it on publish, so it's not forgotten?